test/legacy/getRootStoreTests.ts

```diff
@@ -0,0 +1,9 @@
+import 'jasmine';
+import { isObservableMap } from 'mobx';
+import getRootStore from '../../src/getRootStore';
+
+describe('getRootStore', () => {
+    it('returns an ObservableMap', () => {
+        expect(isObservableMap(getRootStore())).toBeTruthy();
+    });
+});
```

Comment:
I thought I deleted this file..must have reappeared when you moved stuff.  I moved this test to globalContextTests since that's where `rootStore` is actually initialized.